documentation/docs/30-advanced/25-errors.md

```diff
@@ -77,7 +77,7 @@ By default, unexpected errors are printed to the console (or, in production, you
 { "message": "Internal Error" }
 ```
 
-Unexpected errors will go through the [`handleError`](hooks#shared-hooks-handleerror) hook, where you can add your own error handling — for example, sending errors to a reporting service, or returning a custom error object.
+Unexpected errors will go through the [`handleError`](hooks#shared-hooks-handleerror) hook, where you can add your own error handling — for example, sending errors to a reporting service, or returning a custom error object. `handleError` is passed the `error` along with the `event` and a `status` and `message`. `message` is just `"Internal Error"` for unforseen errors, in which case the `status` is 500. `status` may also contain other values such as 404 (page not found) or 415 for actions (wrong content-type), in which case a more specific but still safe `message` is provided.
```

Comment:
"Unexpected errors" is too vague

Reply:
What is a better term? (the phrase was there previously, too)

Reply:
I would say, "Errors thrown without using [`error`](...)"

Reply:
What "unexpected" means is specified in the paragraph above it, and the section is also called "unexpected errors" - I think it's fine as-is.

Reply:
Ah, yeah, that's probably fine in this location. It wasn't showing the paragraph above in the diff. It might not be as clear on the previous page though. I'll comment there